presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveCoercion.java

```diff
@@ -116,6 +120,24 @@
                         "STORED AS PARQUET");
     }
 
+    private static HiveTableDefinition.HiveTableDefinitionBuilder avroTableDefinitionBuilder()
+    {
+        return HiveTableDefinition.builder("avro_hive_coercion")
+                .setCreateTableDDLTemplate("" +
+                        "CREATE TABLE %NAME%(" +
+                        "    int_to_bigint              INT," +
```

Comment:
Looks like Avro is pretty limited in terms of coercions. We should create another patch that documents these limitations? What do you think @rschlussel?